src/DefaultEditor.js

```diff
@@ -13,15 +13,17 @@ import {
 } from './default_panels';
 
 const DefaultEditor = ({localize: _}) => (
-  <PanelMenuWrapper>
-    <GraphCreatePanel group={_('Graph')} name={_('Create')} />
-    <StyleTracesPanel group={_('Style')} name={_('Traces')} />
-    <StyleLayoutPanel group={_('Style')} name={_('Layout')} />
-    <StyleNotesPanel group={_('Style')} name={_('Notes')} />
-    <StyleAxesPanel group={_('Style')} name={_('Axes')} />
-    <StyleLegendPanel group={_('Style')} name={_('Legend')} />
-    <StyleColorbarsPanel group={_('Style')} name={_('Color Bars')} />
-  </PanelMenuWrapper>
+  <Fragment>
+    <PanelMenuWrapper>
```

Comment:
is Fragment needed there?
If I understand correctly, Fragment's used to group elements in place of wrapping them in a div, but I don't see why it'd be useful here..?

Reply:
Whoops that's left over from when I was exploring ways of adding the modal :)